tensorboard/data/server/pip_package/setup.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import setuptools
+
+import tensorboard_data_server
+
+
+setuptools.setup(
+    name="tensorboard_data_server",
+    version=tensorboard_data_server.__version__.replace("-", ""),
+    description="Fast data loading for TensorBoard",
+    long_description="Fast data loading for TensorBoard",
+    url="https://github.com/tensorflow/tensorboard/tree/master/tensorboard/data/server",
+    author="Google Inc.",
```

Comment:
nit: Google LLC is more correct but I bet we have made this mistake elsewhere too.